src/CheckoutSdk/Reconcilation/PayoutStatement.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Checkout.Common;
+using Checkout.Common.Extensions;
+using System;
+
+namespace Checkout.Reconciliation
+{
+    public sealed class PayoutStatement : IEquatable<PayoutStatement>
+    {
+        public string Id { get; set; }
+
+        public string Status { get; set; }
+
+        public double Currency { get; set; }
```

Comment:
I think this is an `enum`

Reply:
Yes, it looks like but on the documentation and in java it say string. Even if is string response it will be correct to cast to Enum, but there is no place mention which values should it takes.